docs/src/pages/[platform]/connected-components/storage/storagemanager/props.ts

```diff
@@ -19,10 +19,15 @@ const displayText = {
 const eventHandler = `(file: {key: string}) => void;`;
 
 export const STORAGE_MANAGER = [
+  {
+    name: `path`,
+    description: 'Path in s3 to put the file under',
+    type: `string | (input: { identityId: string | undefined }) => string`,
+  },
   {
     name: `accessLevel`,
     description:
-      'Access level for files in Storage. See https://docs.amplify.aws/lib/storage/configureaccess/q/platform/js/',
+      'Deprecated, use `path` instead. Access level for files in Storage. See https://docs.amplify.aws/lib/storage/configureaccess/q/platform/js/',
```

Comment:
This will need to get fixed at the type level in another PR, but we shouldn't be linking to this old url anymore. 
Should be changed to https://docs.amplify.aws/javascript/build-a-backend/storage/configure-access/